src/nlv_async_worker.h

```diff
@@ -79,12 +79,43 @@ class NLVPrimitiveReturnWorker : public NLVAsyncWorker<HandleType>
 };
 
 /**
+ * Worker that returns a primitive to javascript
+ */
+#define NLV_STRING_RETURN_WORKER(Method, HandleType, PrimitiveType)  \
```

Comment:
out of my own ignorance, why is this macro named `NVL_STRING*` and we seem to allow passing any type through `PrimitiveType`?